docs/notebooks/flax_intro.ipynb

```diff
@@ -499,7 +499,8 @@
       "outputs": [],
       "source": [
         "x = jnp.ones((1, 2))\n",
-        "ys, model = Dense.partial(features=4).create(random.PRNGKey(0), x)\n",
+        "ys, initial_params = Dense.partial(features=4).init(random.PRNGKey(0), x)\n",
+        "model = nn.Model(Dense, initial_params)\n",
```

Comment:
This isn't correct, you need to first do:

```
model_def = Dense.partial(feature=4)
```

And then use `model_def` in both the calls to `init` and the `nn.Model` constructor.

Reply:
Oh, yes! Done.